servers/dsub/Dockerfile

```diff
@@ -1,3 +1,12 @@
+FROM openjdk
+
+WORKDIR /job-manager
+
+ADD ./ /job-manager
+
+RUN /bin/bash -c scripts/rebuild_swagger.sh
+RUN scripts/await_md5_match.sh /job-manager/ui/src/app/shared/model/.jobs.yaml.md5
```

Comment:
You probably don't need this, since it only exists to delay startup until the swagger is rebuilt (which is guaranteed to have already happened here)

Reply:
- Specifically, you probably don't need the `RUN scripts/await_md5_match.sh /job-manager/ui/src/app/shared/model/.jobs.yaml.md5` line 
- You definitely *do* need the `RUN /bin/bash -c scripts/rebuild_swagger.sh` line!

Reply:
Oh, I see what you mean. I didn't realize the functionality of this script. I think we can take it out. Same for other images